.github/workflows/dev-db.yml

```diff
@@ -18,23 +18,16 @@ on:
         required: true
         default: platsrv-registry-flyway-dev-build
 
-defaults:
-  run:
-    working-directory: ./db
+# Is this needed?
+#defaults:
+#  run:
+#    working-directory: ./db
```

Comment:
The `working-directory: .` is there because of a global setting up above:
`defaults:
  run:
    working-directory: ./web`
Without unsetting that, the `oc` command fails.  An alternative would be to move the working-directory bit into the test and build jobs.